deployment/network-operator/README.md

```diff
@@ -458,7 +458,8 @@ containerResources:
 | `rdmaSharedDevicePlugin.version`            | string | `v1.3.2`                      | RDMA Shared device plugin version                                                                       |
 | `rdmaSharedDevicePlugin.imagePullSecrets`   | list   | `[]`                          | An optional list of references to secrets to use for pulling any of the RDMA Shared device plugin image |
 | `rdmaSharedDevicePlugin.resources`          | list   | See below                     | RDMA Shared device plugin resources                                                                     |
-| `rdmaSharedDevicePlugin.containerResources` | []     | not set                       | Optional [resource requests and limits](#container-resources) for the `rdma-shated-dp` container        |
+| `rdmaSharedDevicePlugin.containerResources` | []     | not set                       | Optional [resource requests and limits](#container-resources) for the `rdma-shared-dp` container        |
+| `rdmaSharedDevicePlugin.useCdi` | bool | `false`                                     | Enable Container Device Interface (CDI) mode. NOTE: NVIDIA Network Operator does not configure container runtime to enable CDI |
```

Comment:
nit: can you align indentation ? (i think markdown plugin for vscode supports auto alignment)

Reply:
Done